src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -3254,9 +2998,9 @@ static const StreamListItem streams_list[] = {
 	create_stream_list_item<MavlinkStreamBatteryStatus>(),
 	create_stream_list_item<MavlinkStreamSmartBatteryInfo>(),
 	create_stream_list_item<MavlinkStreamHighresIMU>(),
-	create_stream_list_item<MavlinkStreamScaledIMU>(),
-	create_stream_list_item<MavlinkStreamScaledIMU2>(),
-	create_stream_list_item<MavlinkStreamScaledIMU3>(),
+	create_stream_list_item<MavlinkStreamScaledIMU<0> >(),
```

Comment:
I've been putting all of these behind ifdefs so that we can easily do stripped down builds with minimal support.

```suggestion
#if defined(SCALED_IMU_HPP)
	create_stream_list_item<MavlinkStreamScaledIMU<0> >(),
```